Sources/ComposableCoreLocation/Live.swift

```diff
@@ -16,264 +16,247 @@ extension LocationManager {
-    create: { id in
-      Effect.run { subscriber in
-        let manager = CLLocationManager()
-        var delegate = LocationManagerDelegate(subscriber)
-        manager.delegate = delegate
-
-        dependencies[id] = Dependencies(
-          delegate: delegate,
-          manager: manager,
-          subscriber: subscriber
-        )
+  public static var live: Self {
+    let manager = CLLocationManager()
 
-        return AnyCancellable {
-          dependencies[id] = nil
-        }
-      }
-    },
-    destroy: { id in
-      .fireAndForget {
-        dependencies[id]?.subscriber.send(completion: .finished)
-        dependencies[id] = nil
-      }
-    },
-    dismissHeadingCalibrationDisplay: { id in
-      .fireAndForget {
-        #if os(iOS) || os(macOS) || os(watchOS) || targetEnvironment(macCatalyst)
-          dependencies[id]?.manager.dismissHeadingCalibrationDisplay()
-        #endif
-      }
-    },
-    heading: { id in
-      #if os(iOS) || os(watchOS) || targetEnvironment(macCatalyst)
-        return dependencies[id]?.manager.heading.map(Heading.init(rawValue:))
-      #else
-        return nil
-      #endif
-    },
-    headingAvailable: {
-      #if os(iOS) || os(macOS) || os(watchOS) || targetEnvironment(macCatalyst)
-        return CLLocationManager.headingAvailable()
-      #else
-        return false
-      #endif
-    },
-    isRangingAvailable: {
-      #if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
-        return CLLocationManager.isRangingAvailable()
-      #else
-        return false
-      #endif
-    },
-    location: { id in dependencies[id]?.manager.location.map(Location.init(rawValue:)) },
-    locationServicesEnabled: CLLocationManager.locationServicesEnabled,
-    maximumRegionMonitoringDistance: { id in
-      #if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
-        return dependencies[id]?.manager.maximumRegionMonitoringDistance ?? CLLocationDistanceMax
-      #else
-        return CLLocationDistanceMax
-      #endif
-    },
-    monitoredRegions: { id in
-      #if os(iOS) || os(macOS) || targetEnvironment(macCatalyst)
-        return Set(dependencies[id]?.manager.monitoredRegions.map(Region.init(rawValue:)) ?? [])
-      #else
-        return []
-      #endif
-    },
-    requestAlwaysAuthorization: { id in
-      .fireAndForget {
-        #if os(iOS) || os(macOS) || os(watchOS) || targetEnvironment(macCatalyst)
-          dependencies[id]?.manager.requestAlwaysAuthorization()
-        #endif
-      }
-    },
-    requestLocation: { id in
-      .fireAndForget { dependencies[id]?.manager.requestLocation() }
-    },
-    requestWhenInUseAuthorization: { id in
-      .fireAndForget {
-        #if os(iOS) || os(macOS) || os(watchOS) || targetEnvironment(macCatalyst)
-          dependencies[id]?.manager.requestWhenInUseAuthorization()
-        #endif
-      }
-    },
-    requestTemporaryFullAccuracyAuthorization: { id, purposeKey in
-      .fireAndForget {
+    return Self(
```

Comment:
And use the initializer to set up all the data at once rather than mutate the value. Actually caught quite a few endpoints we weren't setting 😬